lib/metrics.js

```diff
@@ -32,7 +29,7 @@ const checkAudits = audits => Object.keys(audits).forEach(key => {
     throw new Error(`${debugString} Audit key: ${key}`);
 });
 
-function prepareData(res) {
+function prepareData(res, desiredMetrics = []) {
```

Comment:
It's not possible to have `desiredMetrics ` empty array due to https://github.com/paulirish/pwmetrics/pull/71/files#diff-6d186b954a58d5bb740f73d84fe39073R40